packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -1671,6 +1674,130 @@ class _AnimatedRotationState extends ImplicitlyAnimatedWidgetState<AnimatedRotat
   }
 }
 
+/// Widget which automatically transitions the child's
+/// offset relative to its normal position whenever the given offset changes.
+///
+/// The translation is expressed as an [Offset] scaled to the child's size. For
+/// example, an [Offset] with a `dx` of 0.25 will result in a horizontal
+/// translation of one quarter the width of the child.
+///
+/// {@tool dartpad --template=stateful_widget_scaffold}
```

Comment:
The `stateful_widget_scaffold` is not expecting you to define classes and thus the pre-submit test errors you are seeing.

Look at other examples to see that you only need the stuff inside your State class to use that template.

Reply:
Thanks! Now it works

Reply:
I wonder now why Linux docs test fails